zipkin-ui/js/component_data/default.js

```diff
@@ -3,11 +3,24 @@ import $ from 'jquery';
 import queryString from 'query-string';
 import {traceSummary, traceSummariesToMustache} from '../component_ui/traceSummary';
 
+export function parseApiQuery(windowLocationSearch) {
```

Comment:
Is there a better name for this function? It seems to like does more than just parse the query?